python/nvfuser/testing/narrow_precision.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
```

Comment:
Helper functions can be shared if placed in `https://github.com/NVIDIA/Fuser/blob/main/tests/python/utils/utils.py`. Is there a need for it to be in the package itself?

Reply:
I'm leaning towards shipping that, thinking about how thunder might need those utils as well.

Reply:
It feels odd for Thunder to reuse these functions from nvfuser instead of duplicating them. It is more difficult to change helper functions if Thunder depends on them. Also, YAGNI.

Reply:
fair. Let me move this back to our utils folder then.